independent-publisher-connectors/PaySpace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4476 @@
+                        "type": "string",
+                        "description": "AddressLine1"
+                      },
+                      "AddressLine2": {
+                        "type": "string",
+                        "description": "AddressLine2"
+                      },
+                      "AddressLine3": {
+                        "type": "string",
+                        "description": "AddressLine3"
+                      },
+                      "AddressCode": {
+                        "type": "string",
+                        "description": "AddressCode"
+                      },
+                      "AddressCountry": {
+                        "type": "string",
+                        "description": "AddressCountry"
+                      },
+                      "Province": {
+                        "type": "string",
+                        "description": "Province"
+                      },
+                      "UnitNumber": {
+                        "type": "string",
+                        "description": "UnitNumber"
+                      },
+                      "Complex": {
+                        "type": "string",
+                        "description": "Complex"
+                      },
+                      "StreetNumber": {
+                        "type": "string",
+                        "description": "StreetNumber"
+                      },
+                      "SameAsPhysical": {
+                        "type": "boolean",
+                        "description": "SameAsPhysical"
+                      },
+                      "IsCareofAddress": {
+                        "type": "boolean",
+                        "description": "IsCareofAddress"
+                      },
+                      "CareOfIntermediary": {
+                        "type": "string",
+                        "description": "CareOfIntermediary"
+                      },
+                      "SpecialServices": {
+                        "type": "string",
+                        "description": "SpecialServices"
+                      }
+                    }
+                  },
+                  "description": "Address"
+                }
+              }
+            }
+          }
+        }
+      },
+      "patch": {
+        "summary": "Update a single employee record",
+        "description": "Update a single employee record based on the specified EmployeeId.",
+        "operationId": "UpdateASingleEmployeeRecord",
+        "parameters": [
+          {
+            "name": "company-id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "{{company-id}} The Company Id",
+            "x-ms-summary": "{{company-id}}"
+          },
+          {
+            "name": "EmployeeId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "{{EmployeeId}} The EmployeeId"
```

Comment:
@mint-virgil The 'x-ms-summary' property is required. please add.

Reply:
updated in the latest commit